LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2020 MongoDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
same here- I genuinely don't understand how these files ended up here. I left them because someone else appears to have added them and I didn't want to delete anything important. Should I just delete them?

Reply:
These are from branching off master, which has 1 commit that the new-ui-branch doesn't have. It doesn't really matter whether this is here or not since it'll get added when the branches merge